pkg/webhook/resources/setting/validator.go

```diff
@@ -1304,8 +1308,31 @@ func (v *settingValidator) checkStorageNetworkRangeValid(config *storagenetworkc
 		return err
 	}
 	if !network.IP.Equal(ip) {
-		return fmt.Errorf("Range should be subnet CIDR %v", network)
+		return fmt.Errorf("range should be subnet CIDR %v", network)
 	}
+
+	lhnodes, err := v.lhNodeCache.List(metav1.NamespaceAll, labels.Everything())
```

Comment:
As with the case in the controller, if `len(lhNode.Spec.Disks)` is the same for all the LH nodes, then we can just ask the parent daemonset for the desired number of LH manager pods (hence, IPs), without needing to ask K8s for `Everything` in all namespaces.